packages/bls-kms-local/src/__tests__/bls-key-manager.test.ts

```diff
@@ -0,0 +1,81 @@
+import {generateBls12381G2KeyPair} from '@mattrglobal/node-bbs-signatures';
+import {BlsKeyManager} from "../agent/BlsKeyManager";
+import {MemoryKeyStore, MemoryPrivateKeyStore} from '@veramo/key-manager'
+import {IKey} from "@veramo/core";
+import {BlsKeyManagementSystem} from "../agent/BlsKeyManagementSystem";
+
+describe('@sphereon/ssi-sdk-bls-kms-local', () => {
+    let bls: { publicKey: Uint8Array, secretKey: Uint8Array };
+    let kms: BlsKeyManager;
+
+    beforeAll(async () => {
+        bls = await generateBls12381G2KeyPair();
+        kms = new BlsKeyManager({ store: new MemoryKeyStore(),
```

Comment:
why is this kms defined here? for rest you define it in the restAgent.tests and for local you define this in the agent.yaml

Reply:
There is no agent at this point. Please check @veramo/kms-local/src/__tests__/kms-local.test.ts or @veramo/kms-local/src/__tests__/kms-local.test.ts